helper_apps/MediaFileChunker/package.json

```diff
@@ -18,6 +18,6 @@
     "ioredis": "^5.3.1",
     "public-ip": "^6.0.1",
     "uuid": "^9.0.0",
-    "ytdl-core": "git+ssh://git@github.com:khlevon/node-ytdl-core.git#v4.11.3-patch.1"
+    "ytdl-core": "github:GreepTheSheep/node-ytdl-core"
```

Comment:
Probably don't want to take a dependency on this sort of unknown package - it's a security risk for sure.  Might be better for us to fork ytdl-core to aj-archipelago?

Reply:
so i found out the prev patcher that we were using patched it github:khlevon/node-ytdl-core#v4.11.4-patch.2 
i switched to it, does it make sense or should i still fork?